src/DynamoCoreWpf/Views/Core/DynamoView.xaml.cs

```diff
@@ -61,8 +61,9 @@ public partial class DynamoView : Window, IDisposable
         public const string BackgroundPreviewName = "BackgroundPreview";
 
         //The "Packages" and "Get Started" strings needs to be hardcoded due that are hardcoded in the json file (no need localization)
-        private static string GetStartedGuideName = "Get Started";
-        private static string PackagesGuideName = "Packages";
+        internal static string GetStartedGuideName = "Get Started";
+        internal static string PackagesGuideName = "Packages";
+        internal static string OnboardingGuideName = "Packages";
 
```

Comment:
These access modifiers were changed to be accessed in the SurveyPopupWindow class when the link is clicked.